### PR TITLE
Router

### PR DESCRIPTION
**Pull request by sebastianovide**
introducing routing which allows to have an easier way of managing navigation

### CODE REVIEW

**Comment by sebastianovide** [comment]
for #118

File: src/components/LandingPage.js [hunk 1]
```diff
@@ -1,14 +1,18 @@
 import React, { Component } from 'react';
-import menu from '../images/menu.svg';
-import camera from '../images/camera.svg';
-import map from '../images/map.svg';
+// import { BrowserRouter as Router, Route, Link } from "react-router-dom";
```
Comment (joykuotw) [inline src/components/LandingPage.js]: still need it?

File: src/App.js [hunk 2]
```diff
@@ -94,9 +64,7 @@ class App extends Component {
     });
 
     this.setLocationWatcher();
-
-    this.openLandingPage();
-  }
+    }
```
Comment (joykuotw) [inline src/App.js]: indentation is incorrect

File: src/App.js [hunk 3]
```diff
@@ -19,46 +22,13 @@ class App extends Component {
...
-  openModeratorPage = () => {
-    this.setState({ renderPage: () => (<config.ModeratorPage closePage={this.openLandingPage} photos={this.state.photosToModerate}/>) });
-  };
-
   openPhotoPage = (file) => {
     this.setState({
-      renderPage: () => (<PhotoPage location={this.state.location} file={this.state.file} closePage={this.openLandingPage}/>),
       file
     });
+    this.props.history.push('/photo')
```
Comment (joykuotw) [inline src/App.js]: missing semicolon.

**Pull request merged**